t01primeshcflcm_isprime2.py

```diff
@@ -1,12 +1,11 @@
 # Is num a prime number? 
 # A prime number is an integer with exactly 2 different factors, 1 and itself.
-# Can you rectify the following program error?
+# What happens if you add 3 to num? Does num 2 become a prime number?
 
-num = 61
+num = 28
 
+prime = False
+
+3+num= 31
+31= num 2 
 prime = True
-for i in range(2,num): # won't work for 2 which is the only even prime
```

Comment:
this looks correct, but do you need to go over all numbers between 2 and your number?
if n = p*q, i.e non prime, what equations can be drawn on p or q? can you limit them?